assets/sass/components/global/_googlesitekit-view-only-menu.scss

```diff
@@ -0,0 +1,123 @@
+/**
+ * ViewOnlyMenu styles.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-view-only-menu {
+
+		h4 {
+			font-size: 15px;
+			margin: 15px 0;
+		}
+
+		p {
+			color: $c-primary;
+			font-size: 14px;
+		}
+
+		.mdc-button--dropdown {
+
+			.mdc-button__label {
+
+				@media (max-width: $bp-mobileOnly) {
+
+					@include screen-reader-only;
+				}
+			}
+
+			.mdc-button__icon {
+
+				margin-right: 0;
+
+				@media (min-width: $bp-tablet) {
+					height: 16px;
+					margin-right: 8px;
+					width: 16px;
+				}
+			}
+
+			.mdc-button__icon--image {
+				height: inherit;
+				width: inherit;
+			}
+		}
+
+		.mdc-menu {
+
+			width: 320px;
+		}
+
+		.mdc-list-divider {
+			margin: 8px 32px 4px;
+		}
+
+		.googlesitekit-view-only-menu__list-item {
```

Comment:
Please see **Relevant technical choices** in the PR description for the rationale behind this class.